com.unity.formats.fbx/Editor/IExportData.cs

```diff
@@ -123,37 +123,25 @@ IExportOptions exportOptions
             }
         }
 
-        /// <summary>
-        /// Get the property propertyName from object obj using reflection.
-        /// </summary>
-        /// <param name="obj"></param>
-        /// <param name="propertyName"></param>
-        /// <returns>property propertyName as an object</returns>
-        private static object GetPropertyReflection(object obj, string propertyName)
-        {
-            return obj.GetType().GetProperty(propertyName).GetValue(obj, null);
-        }
-
         /// <summary>
         /// Get the timeline clip from the given editor clip using reflection.
         /// </summary>
         /// <param name="editorClip"></param>
         /// <returns>the timeline clip or null if none</returns>
         private static TimelineClip GetTimelineClipFromEditorClip(object editorClip)
         {
-            object clip = GetPropertyReflection(editorClip, "clip");
+            object clip = editorClip.GetType().GetProperty("clip").GetValue(editorClip, null);
             return clip as TimelineClip;
         }
 
         /// <summary>
-        /// Get the GameObject that the editor clip is bound to in the timeline using reflection.
+        /// Get the GameObject that the clip is bound to in the timeline.
         /// </summary>
-        /// <param name="editorClip"></param>
-        /// <returns>The GameObject bound to the editor clip or null if none.</returns>
-        private static GameObject GetGameObjectBoundToEditorClip(object editorClip)
+        /// <param name="timelineClip"></param>
+        /// <returns>The GameObject bound to the timeline clip or null if none.</returns>
+        private static GameObject GetGameObjectBoundToTimelineClip(TimelineClip timelineClip)
         {
-            var timelineClip = GetTimelineClipFromEditorClip(editorClip);
-            object parentTrack = timelineClip.parentTrack;
+            object parentTrack = timelineClip.GetParentTrack();
```

Comment:
```
	message: Library/PackageCache/com.unity.formats.fbx@e3e0e2667cb2-1618952766000/Editor/IExportData.cs(144,47): error CS1061: 'TimelineClip' does not contain a definition for 'GetParentTrack' and no accessible extension method 'GetParentTrack' accepting a first argument of type 'TimelineClip' could be found (are you missing a using directive or an assembly reference?)
```

Reply:
Do we need to depend on a newer version of timeline?

Reply:
Looks like we need to bump the dependency in package.json to 1.5.4 (which we can since we ditched support for Unity 2018).

Reply:
1.5.4 seems to be broken in 2020.1

Reply:
1.5.2 seems to be working though (from a quick test)

Reply:
Ugh. Yeah, please check what the oldest timeline is that works on all versions.